From a8b99821f16346f1ae4f27b87d3a50211eff3bda Mon Sep 17 00:00:00 2001
From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Wed, 19 Mar 2014 15:49:23 +0100
Subject: [PATCH 025/110] ARM: dts: i.MX53: Enable CODA7541 VPU

This IP module is always present and has no external connections.
There is no reason to disable it in the device tree.

Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
Signed-off-by: Lucas Stach <l.stach@pengutronix.de>
Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 arch/arm/boot/dts/imx53.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx53.dtsi b/arch/arm/boot/dts/imx53.dtsi
index 04815c6..39ea33a 100644
--- a/arch/arm/boot/dts/imx53.dtsi
+++ b/arch/arm/boot/dts/imx53.dtsi
@@ -728,7 +728,6 @@
 				         <&clks IMX5_CLK_VPU_GATE>;
 				clock-names = "per", "ahb";
 				iram = <&ocram>;
-				status = "disabled";
 			};
 		};
 
-- 
1.9.2

